src/nav-soknad/containers/containerUtils.tsx

```diff
@@ -11,44 +11,44 @@ import {
 } from "../../digisos/redux/soknad/soknadActions";
 import {loggAdvarsel, loggFeil} from "../../digisos/redux/navlogger/navloggerActions";
 
-export const erPaStegEnOgValgtNavEnhetErUgyldig = (stegnummer: number, valgtNavEnhet: NavEnhet | undefined): boolean => {
-    return stegnummer === 1 && (!valgtNavEnhet || (valgtNavEnhet && valgtNavEnhet.isMottakMidlertidigDeaktivert) || (valgtNavEnhet && !valgtNavEnhet.enhetsnr))
+export const erPaStegEnOgValgtNavEnhetErUgyldig = (stegnummer: number, valgtNavEnhet: NavEnhet | null): boolean => {
+    return stegnummer === 1 && (!valgtNavEnhet || valgtNavEnhet.isMottakDeaktivert || valgtNavEnhet.isMottakMidlertidigDeaktivert);
 };
 
 export const valgtNavKontorErGyldig = (valgtNavKontor: NavEnhet | undefined): boolean => {
-    return valgtNavKontor && !valgtNavKontor.isMottakMidlertidigDeaktivert ? true : false;
+    return (valgtNavKontor !== undefined && !valgtNavKontor.isMottakDeaktivert && !valgtNavKontor.isMottakMidlertidigDeaktivert);
 };
 
 export const valgtNavKontorErGyldigMenMottakErMidlertidigDeaktivert = (valgtNavKontor: NavEnhet | undefined): boolean => {
-    return valgtNavKontor && valgtNavKontor.isMottakMidlertidigDeaktivert ? true : false
+    return (valgtNavKontor !== undefined && !valgtNavKontor.isMottakDeaktivert && valgtNavKontor.isMottakMidlertidigDeaktivert)
 };
 
-export const responseIsOfTypeListAndContainsAtleastOneObject = (response: any) => {
-    if(response && typeof response !== "string" && (response as NavEnhet[])[0]){
-        return true;
-    } else {
-        return false;
+export const responseIsOfTypeNavEnhet = (response: any) => {
+    if (response == undefined || typeof response === "string") {
```

Comment:
Vil tro at det skal være tre likhetstegn for å unngå "Compiled with warnings"

Reply:
Det hadde du rett i 😄